src/client/Microsoft.Identity.Client/Internal/Requests/Silent/BrokerSilentStrategy.cs

```diff
@@ -99,12 +99,8 @@ private async Task<MsalTokenResponse> SendTokenRequestToBrokerAsync()
                     throw new MsalUiRequiredException(msalTokenResponse.Error, msalTokenResponse.ErrorDescription);
                 }
 
-                throw MsalServiceExceptionFactory.FromBrokerResponse(msalTokenResponse.Error,
-                                                     MsalErrorMessage.BrokerResponseError + msalTokenResponse.ErrorDescription,
-                                                     string.IsNullOrEmpty(msalTokenResponse.SubError) ?
-                                                     MsalError.UnknownBrokerError : msalTokenResponse.SubError,
-                                                     msalTokenResponse.CorrelationId,
-                                                     msalTokenResponse.HttpResponse);
+                throw MsalServiceExceptionFactory.FromBrokerResponse(msalTokenResponse,
```

Comment:
Isn't this common code for all brokers? Why not let all these properties as they are?